### DISCUSSION TITLE
Code completion works, but chat just spins the progress circle indefinitely

### DISCUSSION THREAD

**Discussion by 2picus** [rjmacarthy/twinny#238] [Q&A]
I have Twinny installed on Ubuntu Linux 22.04, and have installed the codellama:7b-instruct and codellama:7b-code providers via Ollama.  I have the Chat and Fill-in-middle providers set in the extension.  Inline suggestions work now, but when I enter a chat message, it just spins the progress circle and never completes.  What could be going on here?

**Comment by secretgspot** [comment]
for me similar, no errors anywhere, spinner keeps spinning, no autocompletion either

**Comment by truongvinht** [comment]
Hello, I also noticed it this morning (EU). So I checked the changes and noticed that the latest version seems to break this feature. My workaround so far is using an older version of twinny: e.g. code --install-extension rjmacarthy.twinny@3.11.17 worked for me (until this bug is solved in latest version or we get some feedback)

**Reply by 2picus** [reply]
Thanks, that did fix it for me.  I will convert this discussion to an issue so it can hopefully get addressed.

**Comment by rjmacarthy** [comment]
Hello, please could you confirm the settings you are using for chat provider? 

Edit:  I think I found the issue and just released a new version.  Please check and report back.

Thanks.
